README.md

```diff
@@ -152,6 +222,15 @@ The script accepts the following parameters as environment variables:
 - LOG_TO_FILE: A flag indicating if the logs should be sent to a file. Optional. Default value: true.
 - LOG_DIR: The path to a directory where the log file will be searched for or created if it does not exist. Optional. Default value: "./log".
 - LOG_FILE_NAME: The name of the file where the logs will be stored. Optional. Default value: "sth_app.log".
+- SERVICE_PREFIX: The prefix to be added to the service for the creation of the databases. More information below. Optional. Default value: "sth".
```

Comment:
I thinks it is better "DB_PREFIX", since we are going to prefix the database name, independently of how it is created.